zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/znode/groupacl/X509ZNodeGroupAclProvider.java

```diff
@@ -59,6 +59,11 @@
  *    "open read access": this feature concerns read access only. Add (world, anyone r) to all
  *          newly-written znodes whose path prefixes are given in the znode group acl config
  *          (comma-delimited, multiple such prefixes are possible).
+ *    "connection filtering": If the server is a dedicated server that only serves one resource,
```

Comment:
"resource" in this sense is confusing. Let's refrain from using that term. In this case, I think "domain" might be a better term to use here throughout (vs namespace). So something like "single-domain access" might be more appropriate instead of introducing new terms like namespace.